common/templates/context.go

```diff
@@ -657,11 +657,17 @@ func baseContextFuncs(c *Context) {
 	c.addContextFunc("getPinCount", c.tmplGetChannelPins(true))
 	c.addContextFunc("getRole", c.tmplGetRole)
 	c.addContextFunc("getThread", c.tmplGetThread)
+
+	// thread functions
 	c.addContextFunc("createThread", c.tmplCreateThread)
 	c.addContextFunc("deleteThread", c.tmplDeleteThread)
 	c.addContextFunc("addThreadMember", c.tmplThreadMemberAdd)
 	c.addContextFunc("removeThreadMember", c.tmplThreadMemberRemove)
 
+	// forum functions
+	c.addContextFunc("createForumPost", c.tmplCreateForumPost)
+	c.addContextFunc("removeForumPost", c.tmplRemoveForumPost)
```

Comment:
`deleteForumPost` feels better (create > delete, add > remove, give > take logic), also no need for additional `c.tmplRemoveForumPost` why not just `c.tmplDeleteThread`

Reply:
That makes sense, I renamed it "deleteForumPost". I also removed tmplRemoveForumPost in favor of reusing tmplDeleteThread (added a comment about being it able to delete both types of threads like you suggested).